sql-parser/build.gradle

```
@@ -13,6 +13,7 @@ configurations {
 }
 
 dependencies {
+    compile project(':shared')
```

```suggestion
    implementation project(':shared')
```

`compile` is deprecated according to gradle docs.